orcid-persistence/src/main/resources/db/updates/update-external-identifiers-unique-constraint-hsqldb-safe.xml

```diff
@@ -0,0 +1,21 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+	
+    <changeSet id="UPDATE_EXTERNAL_IDENTIFIER_UNIQUE_CONSTRAINT" author="Angel Montenegro" dbms="postgresql">
+        <preConditions onFail="MARK_RAN">
```

Comment:
So @bobcaprice this precondition returns 5 in PROD, so, it means we are safe right? the script will be marked as `ran`

Reply:
yes @amontenegro